backend/Modules/Users/HitMeApp.UsersTests/Cryptography/CryptoTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using HitMeApp.Users.Cryptography;
+using NUnit.Framework;
+
+namespace HitMeApp.UsersTests.Cryptography
```

Comment:
To follow the general convention the project should've been called: `HitMeApp.Users.Tests`

Reply:
Done